cfme/containers/container.py

```diff
@@ -58,3 +58,7 @@ def get_detail(self, *ident):
         """
         self.load_details(refresh=True)
         return details_page.infoblock.text(*ident)
+
+    @staticmethod
+    def get_names():
```

Comment:
Yea, makes sense. A bit shortsighted of me. Will see how I can make it more pagination proof